packages/core/src/components/graphs/scatter.ts

```diff
@@ -166,13 +162,11 @@ export class Scatter extends Component {
 
 	styleCircles(selection: Selection<any, any, any, any>, animate: boolean) {
 		// Chart options mixed with the internal configurations
-		const options = this.model.getOptions();
+		const options = this.getOptions();
```

Comment:
again, why are we removing `model` here?

Reply:
allows components to have their own styles (that are not global to other graphs)